web/db/migrate/20150122220754_create_articles.rb

```diff
@@ -1,10 +0,0 @@
-class CreateArticles < ActiveRecord::Migration
```

Comment:
Oh yeah I totally forgot to bring you up to speed with that.  The tutorial had the resource named "Article".  In an attempt to understand all the magic that was going on, I challenged myself to rename the resource to "Blog"  Unfortunately Blog is a keyword in Ruby so I had to change it to "Blogg" lmao.   I did it so I could see all the places I needed to change the name from "Article" to "Blogg" so I could understand it better